dashboard/test/models/user_test.rb

```diff
@@ -78,6 +78,43 @@ class UserTest < ActiveSupport::TestCase
     assert_equal teachers[0].school_info, teachers[1].school_info
   end
 
+  test 'update_school_info with specific school overwrites user school info' do
+    user = create :teacher, school_info: create(:school_info)
```

Comment:
note: this can be `create :teacher, :with_school_info`

See [with_school_info teacher trait](https://github.com/code-dot-org/code-dot-org/blob/4ff1a52a142e8808036d81f6beaa3dca44b16919/dashboard/test/factories/factories.rb#L64)